webapp/app/elster_client/elster_client.py

```diff
@@ -256,6 +256,21 @@ def _set_names_for_merkzeichen(adapted_form_data):
 
     adapted_form_data['person_b_has_merkzeichen_g'] = adapted_form_data.pop('person_b_gehbeh', None)
 
+    merkzeichen_person_a = [
+        adapted_form_data.get('person_a_disability_degree'),
+        adapted_form_data.get('person_a_has_merkzeichen_bl'),
+        adapted_form_data.get('person_a_has_merkzeichen_g'),
+    ]
+
+    merkzeichen_person_b = [
+        adapted_form_data.get('person_b_disability_degree'),
+        adapted_form_data.get('person_b_has_merkzeichen_bl'),
+        adapted_form_data.get('person_b_has_merkzeichen_g'),
+    ]
+
+    adapted_form_data['person_a_requests_pauschbetrag'] = any(merkzeichen_person_a)
+    adapted_form_data['person_b_requests_pauschbetrag'] = any(merkzeichen_person_b)
+
```

Comment:
This throw-away-code, right? Why did you add it? I don't think we need it at all, honestly. We simply never request a pauschbetrag until the step is implemented. If you still want it to stay, please add a todo comment or I will most certainly overlook this.

Reply:
In line 243 we have exactly the comment you are looking for. You are totally right but I did not want this to either break the acceptance tests nor be open until we add all the steps. It was not such a big task and I thought this will make integration much easier.

Reply:
Ah..okay..I see. I don't think it would make problems in theacceptance tests, but it's fine to leave it. :)

Reply:
You are right. you would just not send the merkzeichen fields to Elster, correct:)